presto-tests/src/main/java/com/facebook/presto/tests/TestingPrestoClient.java

```diff
@@ -193,6 +193,9 @@ else if (type instanceof VarcharType) {
         else if (VARBINARY.equals(type)) {
             return value;
         }
+        else if (type.toString().equals("ObjectId")) { // So there's no circular dependency (presto-mongodb -> presto-tests -> presto-mongodb)
```

Comment:
I think this is related to #5575. We need a way to know if arbitrary types should be treated as binary or text.